ui/packages/catalog-ui-search/src/main/webapp/js/router/routes-dev.js

```diff
@@ -0,0 +1,10 @@
+this._compilation.compiler.options.mode === 'development' ? ({
```

Comment:
❓ Why not check `process.env.NODE_ENV !== 'production'` like what we do in [ApplicationSetup.js](https://github.com/codice/ddf/blob/4eebf3ce02dc7cc92650e0e71141b971f02488e3/ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js#L23-L28)

Reply:
This happens during a loader and it doesn't have that available unfortunately.